fla/ops/kda/combined_chunk.py

```
@@ -0,0 +1,348 @@
+
+@torch.compiler.disable
+def combined_chunk_kda(
+    q: torch.Tensor,
+    k: torch.Tensor,
+    v: torch.Tensor,
+    g: torch.Tensor,
+    beta: torch.Tensor,
+    q_conv1d_weight: torch.Tensor,
+    k_conv1d_weight: torch.Tensor,
+    v_conv1d_weight: torch.Tensor,
+    scale: float = None,
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)